Mathlib/Data/Matroid/Loop.lean

```diff
@@ -239,9 +239,9 @@ lemma comap_isLoop_iff {M : Matroid β} {f : α → β} : (M.comap f).IsLoop e 
 lemma loopyOn_isLoop_iff {E : Set α} : (loopyOn E).IsLoop e ↔ e ∈ E := by
   simp [isLoop_iff, loops]
 
-lemma eq_loopyOn_iff_loops {E : Set α} : M = loopyOn E ↔ M.loops = E ∧ M.E = E :=
-  ⟨fun h ↦ by rw [h, loops]; simp, fun ⟨h,h'⟩ ↦
-    by rw [← h', ← closure_empty_eq_ground_iff, ← loops, h, h']⟩
+lemma eq_loopyOn_iff_loops {E : Set α} : M = loopyOn E ↔ M.loops = E ∧ M.E = E where
```

Comment:
I'm neutral on this change (and happy to see a second opinion).

Reply:
fwiw I like the change.